op-node/rollup/interop/config.go

```diff
@@ -30,55 +33,27 @@ type Config struct {
 }
 
 func (cfg *Config) Check() error {
-	// TODO(#13338): temporary workaround needs both to be configured.
-	//if (cfg.SupervisorAddr == "") != (cfg.RPCAddr == "") {
-	//	return errors.New("must have either a supervisor RPC endpoint to follow, or interop RPC address to serve from")
-	//}
+	if (cfg.SupervisorAddr == "") != (cfg.RPCAddr == "") {
```

Comment:
```suggestion
	if (cfg.SupervisorAddr == "") == (cfg.RPCAddr == "") {
```
Based on the error message and other code, it seems this should be `==`.